tdrs-frontend/src/components/ComboBox/ComboBox.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import comboBox from 'uswds/src/js/components/combo-box'
+
+const ComboBox = ({ sttList }) => {
+  const selectRef = useRef()
+  useEffect(() => {
+    // The combo box was not rendering as a combo box without this line
+    comboBox.init()
+    // This solved the issue when tabbing through the form on EditProfile,
+    // a selection was automatically being made on the first option
+    selectRef.current.value = ''
+  })
+  return (
+    <label className="usa-label" htmlFor="sttList">
+      Associated State, Tribe, or Territory
+      <div className="usa-combo-box">
+        <select className="usa-select" name="stt" id="stt" ref={selectRef}>
+          {sttList.map((stt) => (
+            <option
+              className="sttOption"
+              key={stt.id}
+              value={stt.name.toLowerCase()}
+            >
+              {stt.name}
+            </option>
+          ))}
+        </select>
+      </div>
+    </label>
+  )
+}
+
+ComboBox.propTypes = {
+  sttList: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      id: PropTypes.number,
+    })
+  ).isRequired,
```

Comment:
I pulled out the ComboBox component to clean up EditProfile which was becoming really long and complex. I am happy to add it back into the EditProfile if anyone thinks that it is too cumbersome to have it separated.